library/modules/recv_all.py

```diff
@@ -11,4 +15,17 @@ def main(sock):
                 raise socket.error
             data += tmp_data
         except (socket.error, socket.timeout):
-            return data
\ No newline at end of file
+            if data.startswith('png:'):

```

Comment:
Basically sees if it starts with a png: and writes to your disk and not the scouts and so no traces left behind